### PR TITLE
removing black border

### PR DESCRIPTION
**Pull request by jyng**
Removing `bb` from images

@urcades @matildepark

### CODE REVIEW

**Comment by matildepark** [comment]
This removes the border from all links, always.

**Comment by matildepark** [comment]
You want to refactor it so that it applies `bb` only if ytMatch and imgMatch both come in false

**Comment by matildepark** [comment]
You should also test and ensure it actually works that way in your dev environment

File: pkg/interface/chat/src/js/components/lib/message.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ export class Message extends Component {
           ></img>
         );
         return (
```
Comment (jyng) [inline pkg/interface/chat/src/js/components/lib/message.js]: @matildepark my understanding of the code is that it, if imgMatch, it returns the the classes in className. It seems to render that correctly?

image

**Comment by matildepark** [comment]
Shit, you're right, sorry — was a little quick on the draw from my phone.

**Comment by jyng** [comment]
@jtobin updated commit message to be more descriptive.

**Pull request merged**